packages/grafana-ui/src/components/FileDropzone/FileDropzone.tsx

```diff
@@ -1,11 +1,12 @@
 import { css, cx } from '@emotion/css';
-import { uniqueId, isString } from 'lodash';
+import { isString, uniqueId } from 'lodash';
 import React, { ReactNode, useCallback, useState } from 'react';
-import { DropEvent, DropzoneOptions, FileRejection, useDropzone, Accept } from 'react-dropzone';
+import { Accept, DropEvent, DropzoneOptions, FileRejection, useDropzone } from 'react-dropzone';
```

Comment:
it's not a massive issue, but is there a reason you changed the order of these imports?

Reply:
I'm curious too, on first glance though it seems like an alphabetization re-order (maybe done by IDE automatically?) Not sure if we have import order rule set on alphabetization or not (locally for me it does not appear to complain if not alphabetically sorted)

Reply:
I think it was done automatically by my IDE as I did not change the order. I'll check the settings and disable if the case.